.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
```

Comment:
i like the idea of the `pre-commit` configuration but i would propose:

1. move this into a separate PR
2. ask in #sig-dev/#sig-releng about opinions (maybe it's worth to enable globally via our automation)

Reply:
removed.